send-slack-notification/action.yaml

```
@@ -35,11 +60,11 @@ runs:
         token: ${{ inputs.slack-token }}
         payload: |
           channel: "${{ inputs.channel-id }}"
-          text: "*${{ github.workflow }}* failed (attempt ${{ github.run_attempt }})"
-          ${{ github.run_attempt > 1 && format('thread_ts: "{0}"', steps.slack-thread-id.outputs.SLACK_THREAD_ID) || '' }}
+          text: "${{ env.MESSAGE_TEXT }}"
+          ${{ env.SLACK_THREAD_ID != 'none' && format('thread_ts: "{0}"', env.SLACK_THREAD_ID) || '' }}
```

I think this can then be:

```suggestion
          ${{ steps.retrieve-slack-thread-id.outcome == 'success' && format('thread_ts: "{0}"', env.SLACK_THREAD_ID) || '' }}
```